### PR TITLE
diskus 0.5.0 (new formula)

### PR DESCRIPTION
**Pull request by leonklingele**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/diskus.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Diskus < Formula
+  desc "Minimal, fast alternative to 'du -sh'"
```
Comment (colindean) [inline Formula/diskus.rb]: I think having the word "parallel" in here would help explain its speed…

**Comment by leonklingele** [comment]
Description was taken from the diskus’ GH repo

**Comment by LEARAX** [comment]
```
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </113\ B\ \(113\ bytes\)/> expected to be =~
<"107 B (107 bytes)\n">.```
😞

**Comment by leonklingele** [comment]
@LEARAX argh, I assume this is on APFS? Updated to a regex.

**Comment by leonklingele** [comment]
TIL: `diskus` _does_ allow specifying the path to check. Test updated.

**Comment by javian** [comment]
Thanks @leonklingele !

**Pull request closed without merging**